src/Features/Units/UnitSelector/index.tsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * Component to select and display currently selected unit type
+ */
+import * as React from "react"
+import { Button, ButtonGroup } from "reactstrap"
+import { connect } from "react-redux"
+import { bindActionCreators, Dispatch } from "redux"
+
+import { StoreState } from "Shared/constants/store"
+
+import { unitSwitch } from "../actions"
+import { UnitSystem } from "../types"
+
+export interface ReduxProps {
+  system: UnitSystem
+  switchUnits: (system: UnitSystem) => void
+}
+
+const mapStateToProps = ({ unit }: StoreState): Pick<ReduxProps, "system"> => {
+  return {
+    system: unit.system
+  }
+}
+
+const mapDispatchToProps = (dispatch: Dispatch) =>
+  bindActionCreators(
+    {
+      switchUnits: unitSwitch
+    },
+    dispatch
+  )
+
+export class UnitSelectorBase extends React.Component<ReduxProps, object> {
+  public render() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature: 'render' to have a typedef](https://app.codacy.com/app/gmri/Neracoos-1-Buoy-App/pullRequest?prid=4354739)